src/CONST.js

```diff
@@ -202,6 +202,10 @@ const CONST = {
     FEES_URL: `${USE_EXPENSIFY_URL}/fees`,
     CFPB_PREPAID_URL: 'https://cfpb.gov/prepaid',
     STAGING_SECURE_URL: 'https://staging-secure.expensify.com/',
+
+    // Use Environment.getEnvironmentURL to get the complete URL with port number
+    DEV_NEW_EXPENSIFY_URL: 'http://localhost:',
```

Comment:
Hmm i feel like its best placed here otherwise you guys won't be able to test the PR locally. I could move it elsewhere but then my test setup steps for dev would have more steps. Let me know if you feel otherwise.